main/src/mill/main/TokenReaders.scala

```diff
@@ -23,19 +23,20 @@ object Tasks {
   }
 }
 
-class EvaluatorTokenReader[T]() extends mainargs.TokensReader.Constant[mill.eval.Evaluator] {
```

Comment:
Those token readers where needed to be accessible in the past, e.g. in some plugins that provided external modules with commands.

Reply:
But do they need to be accessible directly? I'd expect people to use them via `import mill.main.TokenReaders._`, and use them abstracted behind a `mainargs.TokensReader[Evaluator]` interface, rather than `new mill.main.EvaluatorTokenReader` and using the concrete type directly

Reply:
Yeah, that might work. I overlooked that the implicit `def`s are accessible.